traits/trait_types.py

```diff
@@ -3316,7 +3348,7 @@ def post_setattr(self, object, name, value):
         setattr(object, name + "_", self.mapped_value(value))
 
     def info(self):
-        keys = sorted(repr(x) for x in self.map.keys())
+        keys = sorted(repr(x) for x in self.map)
```

Comment:
There's a difference between `PrefixList` and `PrefixMap` here: we sort the output for `PrefixMap` but not `PrefixList`. Given that we can now rely on ordered dictionaries, I think we should drop the `sorted` here.